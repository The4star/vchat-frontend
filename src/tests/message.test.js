import React from 'react';
import { shallow, mount, render } from 'enzyme';
import Message from '../components/message/message.component';

it('renders a card component', () => {
  expect(shallow(< Message />)).toMatchSnapshot()
})