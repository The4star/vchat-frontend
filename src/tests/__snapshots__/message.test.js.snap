// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a card component 1`] = `ShallowWrapper {}`;
